config/config/sc-config.yaml

```
@@ -424,17 +424,17 @@ thanos:
         cpu: 50m
         memory: 50Mi
       limits:
-        cpu: 500m
-        memory: 200Mi
+        cpu: 2000m
```

I'd rather not have them in milli format to not confuse people
```suggestion
        cpu: "2"
```